resources/views/bread/browse.blade.php

```diff
@@ -246,7 +246,8 @@
                                                     @endif
                                                 @else
                                                     @include('voyager::multilingual.input-hidden-bread-browse')
-                                                    <span>{{ $data->{$row->field} }}</span>
+                                                    {{--<span>{{ $data->{$row->field} }}</span>--}}
```

Comment:
Please delete commented code.  There's no point in leaving it when everything is tracked by git